### ISSUE TITLE
TypeError: AsyncSession.request() got an unexpected keyword argument 'thread'

### ISSUE THREAD

**Issue by reggenomeneiell** [jxlil/scrapy-impersonate#3]
```
File "/home/neiellcare/.cache/pypoetry/virtualenvs/rgdl-spiders--SESLHN9-py3.11/lib/python3.11/site-packages/scrapy_impersonate/handler.py", line 44, in _download_request
    response = await self.client.request(**RequestParser(request).as_dict())  # type: ignore
TypeError: AsyncSession.request() got an unexpected keyword argument 'thread'
```

**Comment by chandu-cpz** [comment]
Just got the same error. (can help in reproducing it ). (I dm'ed you in reddit ) .

**Comment by reggenomeneiell** [comment]
I did a hotfix by removing the these lines https://github.com/jxlil/scrapy-impersonate/blob/master/scrapy_impersonate/parser.py#L95-L97

**Comment by chandu-cpz** [comment]
I can confirm it's working .(removing 95-97 lines) . Thanks

**Comment by reggenomeneiell** [comment]
@jxlil `AsyncSession` doesn't support `thread`. it only works on `Session` for `curl-cffi`

**Comment by jxlil** [comment]
Thanks everyone, the problem has already been resolved in version `1.1.1b2`
